src/stages/kinesis-s3.ts

```diff
@@ -48,10 +51,15 @@ export class FirehoseToS3Stage extends DataStage {
       this.dataStream = new kinesis.Stream(this, "Data Stream", {});
     }
 
-    const destinationsBucketProps = FirehoseDestinationsDefaults.s3Props({
-      ...props.kinesisFirehoseDestinationsS3BucketProps,
-      dataOutputPrefix: props.dataOutputPrefix,
-    });
+    const destinationsBucketProps = overrideProps(
+      {
+        dataOutputPrefix: props.dataOutputPrefix,
+        compression: destinations.Compression.GZIP,
+        bufferingInterval: cdk.Duration.seconds(300),
+        bufferingSize: cdk.Size.mebibytes(5),
+      },
+      props.kinesisFirehoseDestinationsS3BucketProps ?? {},
```

Comment:
Same comment as above. I think we want `dataOutputPrefix` to override what's in `kinesisFirehoseDestinationsS3BucketProps`